test/fake-timers-test.js

```diff
@@ -5036,6 +5036,7 @@ describe("loop limit stack trace", function () {
         });
     });
 
+    // This doesn't really work since we're unable to add an error to all running intervals
```

Comment:
Not totally sure what "this" refers to really. The tests or the `setInteval` function or what.

Reply:
tracking of callbacks triggered by `setInterval`